Frameworks/Core/ERExtensions/Sources/er/extensions/components/javascript/ERXJSToManyRelationshipEditor.java

```diff
@@ -258,6 +258,10 @@ private void pushSelectionsBinding(NSArray selections) {
     private NSArray selectionsFromBindings() {
         if (canGetValueForBinding(Keys.Selections)) {
             NSArray result = (NSArray)valueForBinding(Keys.Selections);
+            if (hasBinding(Keys.SortKey)) {
```

Comment:
Should be moved down into the ```if(result != null) {``` block as sorting a non-existing result does not make sense and even will throw an exception in ERXArrayUtilities.